.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
+
+on: [push]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.caching=true -Dorg.gradle.welcome=never"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Gradle
+        uses: GeyserMC/actions/setup-gradle-composite@master
+        with:
+          setup-java_java-version: 17
+
+      - name: Build Base API
+        run: ./gradlew build
+
+      - name: Publish to Maven Repository
+        if: ${{ success() && github.repository == 'GeyserMC/api' }}
+        run: ./gradlew publish
+        env:
+          ORG_GRADLE_PROJECT_geysermcUsername: ${{ vars.DEPLOY_USER }}
+          ORG_GRADLE_PROJECT_geysermcPassword: ${{ secrets.DEPLOY_PASS }}
+
+      - name: Get Version
+        if: ${{ github.repository == 'GeyserMC/api' }}
+        id: get-version
+        run: |
+          version=$(cat build.gradle | grep -o "version '[0-9\\.]*'" | cut -d" " -f2 | sed "s/'//g")
```

Comment:
this must've been a copy/paste mistake, use what you had in #7 for this part

Reply:
oops, thanks - resolved it now